src/roadmap/release-checklist.md

```diff
@@ -106,7 +106,7 @@ version of our release for testing. Here is the current process to do so:
 
 # Step 3: Kick off Jenkins Jobs
 
-Navigate to https://ci.deis.io/job/workflow-test-release/ and kick off a new job with appropriate build parameters filled out, i.e. `HELM_REMOTE_BRANCH=$WORKFLOW_RELEASE` and `RELEASE=$WORKFLOW_RELEASE_SHORT`
+Navigate to https://ci.deis.io/job/workflow-test-release/ and kick off a new job with the appropriate build parameters filled out, i.e. `WORKFLOW_BRANCH=release-$WORKFLOW_RELEASE`, `WORKFLOW_E2E_BRANCH=release-$WORKFLOW_RELEASE` and `RELEASE=$WORKFLOW_RELEASE_SHORT`.
```

Comment:
Thanks @arschles; my plan is to merge this and then in https://github.com/deis/workflow/pull/363 this step will be rewritten and moved (no longer have to manually fill out params or kick off job.)  The only section actually being wholly preserved will the be the latter change below...